webpack/ForemanTasks/Components/common/ActionButton/ActionButton.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const CancelButton = ({ id, name, cancellable, cancel }) => (
+  <Button
+    bsSize="small"
+    onClick={() => cancel(id, name)}
+    disabled={!cancellable}
+  >
+    {__('Cancel')}
+  </Button>
+);
+
+export const ResumeButton = ({ id, name, resumable, resume }) => (
+  <Button bsSize="small" onClick={() => resume(id, name)} disabled={!resumable}>
+    {__('Resume')}
+  </Button>
+);
+
+export const ActionButton = ({ id, name, availableActions, taskActions }) => {
+  const isResume = availableActions.resumable;
+  if (isResume) {
+    return (
+      <ResumeButton id={id} name={name} resume={taskActions.resume} resumable />
+    );
+  }
+  return (
+    <CancelButton
+      id={id}
+      name={name}
+      cancellable={availableActions.cancellable}
+      cancel={taskActions.cancel}
+    />
+  );
+};
+
+ResumeButton.propTypes = {
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  resume: PropTypes.func.isRequired,
+  resumable: PropTypes.bool.isRequired,
```

Comment:
In the context of `ResumeButton` and `CancelButton`, does it make more sense to call it `onClick` and `disabled`?